handsontable/src/plugins/formulas/utils.js

```diff
@@ -17,3 +21,74 @@ export function isEscapedFormulaExpression(expression) {
 export function unescapeFormulaExpression(expression) {
   return isEscapedFormulaExpression(expression) ? expression.substr(1) : expression;
 }
+
+/**
+ * Checks whether string looks like formula or not. Corresponds to {@link https://hyperformula.handsontable.com/api/globals.html#isformula|HyperFormula's implementation}.
+ *
+ * @param {string} value Checked value.
+ * @returns {boolean}
+ */
+export function isFormula(value) {
```

Comment:
Can I ask to cover the `isFormula` util function with tests?

Reply:
Added init tests within: https://github.com/handsontable/handsontable/pull/10317/commits/f8de204b58316cd6649a69e8ea44f6685c3d8aed